docs/jetstream/configuration.md

```diff
@@ -8,24 +8,24 @@ title: Configuring Jetstream
 There are two ways to customize the results that appear in Experimenter and [partybal] for an experiment.
 
 You can either reference an [outcome] while creating the experiment in the console,
-or file a [jira ticket](https://mozilla-hub.atlassian.net/jira/software/c/projects/DO/boards/269) to collaborate with a data scientist commit a custom configuration to the [jetstream-config] repository.
+or file a [jira ticket](https://mozilla-hub.atlassian.net/jira/software/c/projects/DO/boards/269) to collaborate with a data scientist commit a custom configuration for Jetstream to the [metric-hub] repository.
 
-Outcome definitions are also created in jetstream-config and use the same configuration language.
+Outcome definitions are also created in [metric-hub] and use the same configuration language.
 
 Custom experiment configurations are associated with an experiment by their filename,
 which should match the experiment slug, like `my-experiment-slug.toml`.
 This works for both Normandy and Nimbus slugs. 
 
 [partybal]: https://protosaur.dev/partybal/
 [outcome]: jetstream/outcomes.md
-[jetstream-config]: https://github.com/mozilla/jetstream-config
+[metric-hub]: https://github.com/mozilla/metric-hub/tree/main/jetstream
 
 ## Landing configurations
 
-To add or update a custom configuration, a data scientist will open a pull request against [jetstream-config]. 
+To add or update a custom configuration, a data scientist will open a pull request against [metric-hub]. 
 CI checks will validate the columns, data sources, and SQL syntax. Note that if the experiment has not yet launched, the CI checks will not pass.
-Once CI completes, you may merge the pull request, which will trigger Jetstream to re-run your analysis. 
-No additional review is necessary to land configurations. Results should be available in several hours, depending upon the complexity of the configuration.
+Once CI completes and the pull request gets automatically approved, you may merge the pull request, which will trigger Jetstream to re-run your analysis.
+No additional review is necessary to land configurations as long as no changes are made to [metric definitions that are considered  source of truth](https://github.com/mozilla/metric-hub/tree/main/definitions). These changes will require a review by a data scientist. Results should be available in several hours, depending upon the complexity of the configuration.
```

Comment:
To clarify, this DS review is by someone different from the person who opened the PR?

Reply:
Right.

The way the repo is set up is that it requires reviews before a PR can get merged. However, we still want users to be able to merge jetstream and opmon configs without having to ask someone. So if files get added to only `jetstream/` and `opmon/`, then a github bot will auto approve these PRs so that they can get merged.